Jenkinsfile

```diff
@@ -71,13 +53,14 @@ node {
 			}
 
 			if (isPR || isRc || isMaster) {
-				stage('Deploy') {
+				loggableStage('Deploy') {
 					echo "deploying ${branchName}"
 					bat "npm run compodoc -- -p ./tsconfig.doc.json -e json -d .storybook"
 					bat "npm run build-storybook -- -o \\\\labs2.lucca.local\\c\$\\d\\sites\\lucca-front\\${branchName}"
 				}
 
 				if (isPR) {
+					prNumber = env.BRANCH_NAME.substring(3)

```

Comment:
🤔 ?

Reply:
Déplacé cette ligne là où elle servira, le numéro de PR est nécessaire pour la suite

Reply:
J'ai pas osé toucher au code qui suit, @lucienbertin
Si t'as une meilleure idée @GMouron 😁

Reply:
c'est du code que @GMouron m'a aide a ecrire pour poster un com sur la bonne PR

Reply:
Oui ok, techiquement, c'est mon code,  je crois pas que @lucienbertin était prêt psychologiquement à faire du PowerShell 😂 
Nan en vrai faudrait en faire  un step Jenkins, mais sinon 2 choses :
https://github.com/LuccaSA/Jenkins.SharedLibrary/blob/master/src/fr/lucca/jenkins/BuildInfo.groovy#L52
A priori, env.CHANGE_ID a ce qu'on veut.
Sinon, tu fais un `BuildInfo.gather()` (mais faut importer la classe tout en haut)

Reply:
env.CHANGE_ID it is. Updated